randovania/games/samus_returns/logic_database/Area 4 - Central Caves.txt

```diff
@@ -491,14 +525,17 @@ Extra - asset_id: collision_camera_007
   * Tunnel to Alpha+ Arena Access/Tunnel to Alpha+ Arena
   > Pickup (Missile Tank)
       Morph Ball
+  > Start Point
+      Morph Ball
 
 > Start Point; Heals? False; Default Node
   * Layers: default
   * Extra - start_point_actor_name: ST_Alpha_001_Checkpoint
   > Door to Alpha+ Arena Access
       Any of the following:
           High Jump Boots or Gravity Suit or Space Jump or Super Jump (Beginner) or Unmorph Extend (Intermediate)
-          Infinite Bomb Jump (Intermediate) and Lay Bomb
+          # IBJ
+          Mid-Air Morph (Beginner) and Lay Bomb and Use Spider Ball
```

Comment:
Spider doesn't need Bombs, so nice it to the top group.

Bombs needs either mid air Morph, ibj Intermediate from water, or spring ball.

Reply:
whoops